openmetadata-ui/src/main/resources/ui/src/components/Ingestion/Ingestion.component.tsx

```diff
@@ -95,6 +102,24 @@ const Ingestion: React.FC<IngestionProps> = ({
     setSearchText(searchValue);
   };
 
+  const [list, setList] = useState<Array<IngestionPipeline>>([]);
+  const [servicePermission, setServicePermission] =
+    useState<ServicePermission>();
+
+  const fetchServicePermission = () => {
+    list.forEach(async (item) => {
+      try {
+        const response = await getEntityPermissionByFqn(
+          ResourceEntity.INGESTION_PIPELINE,
+          item.name
+        );
+        setServicePermission((prev) => ({ ...prev, [item.name]: response }));
+      } catch (error) {
+        showErrorToast(error as AxiosError);
+      }
+    });
+  };
```

Comment:
@Ashish8689 , I would recommend using the `Promise.allSettled` here.

Reply:
@Sachin-chaurasiya  ok, will update this.